torch/export/dynamic_shapes.py

```diff
@@ -1011,20 +888,43 @@ def update_symbols(path, tensor, shape):
         def _create_static_dim(tensor, i, value):
             return _StaticDim(str(value), (int,), {"value": value})
 
+        # clean out decorators from user side, or previous export call
```

Comment:
Interesting, so we'll only allow specs to dictate which ones are marked? Should we warn if any of these were set, asking to use specs instead?

Reply:
Ah added a comment in the last PR description line